### DISCUSSION TITLE
Chart values below bottom X-Axis Labels?

### DISCUSSION THREAD

**Discussion by binshadpix** [patrykandpatrick/vico#955] [Questions]
### Question

is it possible to provide Chart values below bottom X-Axis Labels like in the image?
![Screenshot 2024-11-22 at 11 57 16 AM](https://github.com/user-attachments/assets/8406c20e-58ee-40a6-97f0-9d90b4c7b284)

### Vico version(s)

1.12.0

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @binshadpix. How are these extra values supposed to be positioned? Should they be directly below the axis labels?

The values in the image don't seem to be particularly tied to the chart layout. They're only approximately aligned with the axis labels.

**Reply by binshadpix** [reply]
hi Gowsky,
They should be directly below the axis labels.
i have achieved it now by using multiline for x axis label and given the values after /n /n
example:- "$xValue /n/n $percentageValue"

**Reply by Gowsky** [reply]
That works.

If you need more flexibility, then another approach is to create a custom `Axis` based on `HorizontalAxis`. (This is easier in Vico 2, where `HorizontalAxis` can be subclassed.)

Also, extras (Vico 1.14.0+) may come in handy.
